src/main/java/racingcar/step3/Race.java

```diff
@@ -0,0 +1,31 @@
+package racingcar.step3;
+
+import java.util.ArrayList;
+import java.util.List;
+
+class Race {
+    public List<Car> getCars() {
+        return cars;
+    }
+
+    private List<Car> cars = new ArrayList<>();
+
+    public Race(int numberOfCars) {
+        for (int i = 0; i < numberOfCars; i++) {
+            cars.add(new Car());
+        }
+    }
+
+    public void start(int attempts) {
+        System.out.println("실행 결과");
+        for (int i = 0; i < attempts; i++) {
+            for (Car car : cars) {
+                if (i != 0) {
+                    car.move();
+                }
+                ResultView.displayRace(car);
+            }
+            System.out.println(" \n ");
```

Comment:
```suggestion
            }
            ResultView.displayRace(cars);
```

자동차 하나하나 보다는 리스트를 한번에 전달하면 더 좋을 것 같습니다.